apps/hash-graph/lib/graph/src/shared/identifier/mod.rs

```diff
@@ -81,11 +84,14 @@ impl From<EntityVertexId> for GraphElementVertexId {
 // WARNING: This MUST be kept up to date with the enum variants.
 //   We have to do this because utoipa doesn't understand serde untagged:
 //   https://github.com/juhaku/utoipa/issues/320
-impl ToSchema for GraphElementVertexId {
-    fn schema() -> openapi::RefOr<openapi::Schema> {
-        openapi::OneOfBuilder::new()
-            .item(openapi::Ref::from_schema_name("OntologyTypeEditionId"))
-            .item(openapi::Ref::from_schema_name("EntityVertexId"))
-            .into()
+impl ToSchema<'_> for GraphElementVertexId {
+    fn schema() -> (&'static str, openapi::RefOr<openapi::Schema>) {
+        (
+            "GraphElementVertexId",
+            openapi::OneOfBuilder::new()
+                .item(openapi::Ref::from_schema_name("OntologyTypeEditionId"))
```

Comment:
Can you use the trick with calling `schema().0` to avoid the magic strings here?

Reply:
Even better: I was able to derive it: 1ccc2695f81b510a40af5c0b5453237bc78b7669